backend/app/adapter/db/changelog.go

```diff
@@ -33,7 +33,7 @@ FROM "%s";`,
 		return []entity.Change{}, err
 	}
 
-	var changeLog []entity.Change
+	changeLog := []entity.Change{}
```

Comment:
`var` automatically initializes the variable with empty array.

Reply:
mdTest failed when i used var

Reply:
@byliuyang

Reply:
![Screenshot from 2020-04-04 13-27-00](https://user-images.githubusercontent.com/20877996/78421792-134a2200-7678-11ea-8b3e-223ef3bc34d8.png)